.buildkite/scripts/steps/esql_grammar_sync.sh

```diff
@@ -7,19 +7,19 @@ synchronize_lexer_grammar () {
   destination_file="./packages/kbn-esql-ast/src/antlr/esql_lexer.g4"
 
   # Copy the file
-  cp "$source_file" "$destination_file" 
```

Comment:
I don't know why all these were removed, or why the spaces where there to begin with, but I hope this won't be an issue.